src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -39,6 +39,7 @@
   <ItemGroup>
     <Reference Include="NCalc, Version=1.3.8.0, Culture=neutral, PublicKeyToken=973cde3f1cafed03, processorArchitecture=MSIL">
       <HintPath>..\..\packages\ncalc.1.3.8\lib\NCalc.dll</HintPath>
+      <Private>False</Private>
```

Comment:
this should be true actually.

previous was:
```
<ItemGroup>
    <Reference Include="NCalc">
      <HintPath>..\..\..\extern\NCalc\NCalc.dll</HintPath>
      <Private>True</Private>
    <Reference Include="NCalc, Version=1.3.8.0, Culture=neutral, PublicKeyToken=973cde3f1cafed03, processorArchitecture=MSIL">
      <HintPath>..\..\packages\ncalc.1.3.8\lib\NCalc.dll</HintPath>
```

we need to deliver this binary somewhere, just not in two places.

Reply:
Added it. Is it because the dll is expected to be in one NuGet package but not in another one?

Reply:
this should end up in the /nodes folder, not in the root directory.